pydfs_lineup_optimizer/lineup_exporter.py

```diff
@@ -36,3 +37,37 @@ def export(self, filename, render_func=None):
                 row.append(str(lineup.salary_costs))
                 row.append(str(lineup.fantasy_points_projection))
                 lineup_writer.writerow(row)
+
+
+class JSONLineupExporter(LineupExporter):
```

Comment:
LineupExporter is a interface for implementing export to file, so export function should have filename parameter and don't return any value as in base class. I think json export to file isn't very useful because dfs sites doesn't accept json. I think it will be better to move creation of dict with information about lineup to Lineup class as a method as_dict or something like this, so users can get dict with necesarry information and pass it to json.dumps function directly in code.

Reply:
If you have time, could you provide an example of this?